persistent/_timestamp.c

```diff
@@ -27,9 +27,40 @@ static char TimeStampModule_doc[] =
 "$Id$\n";
 
 
+
+#define SCONV ((double)60) / ((double)(0x10000)) / ((double)(0x10000))
+#define TS_BASE_YEAR 1900
+#define TS_MINUTES_PER_DAY 1440
+/* We pretend there are always 31 days in a month; this has us using
+   372 days in a year in some calculations */
+#define TS_DAYS_PER_MONTH 31
+#define TS_MONTHS_PER_YEAR 12
+#define TS_MINUTES_PER_MONTH (TS_DAYS_PER_MONTH * TS_MINUTES_PER_DAY)
+#define TS_MINUTES_PER_YEAR (TS_MINUTES_PER_MONTH * TS_MONTHS_PER_YEAR)
+
+#define TS_PACK_UNSIGNED_INTO_BYTES(v, bytes) do { \
+    *(bytes) = v / 0x1000000;                      \
+    *(bytes + 1) = (v % 0x1000000) / 0x10000;      \
+    *(bytes + 2) = (v % 0x10000) / 0x100;          \
+    *(bytes + 3) = v % 0x100;                      \
+} while (0)
+
+#define TS_UNPACK_UNSIGNED_FROM_BYTES(bytes) (*(bytes) * 0x1000000 + *(bytes + 1) * 0x10000 + *(bytes + 2) * 0x100 + *(bytes + 3))
```

Comment:
Likewise here:  what is the semantic of the return value?

Reply:
It's the inverse of `TS_PACK_UNSIGNED_INTO_BYTES`. I documented that in the *TimeStamp* comment, but I can document it here as well.

Reply:
What are the semantics of the returned value?  We are using it for something specific, not random unsigned (what length?) values.

Reply:
Fair question. Hopefully the docs I added address that.

Reply:
And I changed the name from `_UNSIGNED_` into `_UINT32_` to be more specific about the expected range of the input/output.